### PR TITLE
Use new streamlog interface

### PR DESCRIPTION
**Pull request by rete**
BEGINRELEASENOTES
- Use new streamlog interface for logging initialization
- Fixed usage of streamlog using pre-processor macros provided by streamlog

ENDRELEASENOTES

Requires https://github.com/iLCSoft/iLCUtil/pull/14

### CODE REVIEW

File: source/Tests/include/TestUtilities.hh [hunk 1]
```diff
@@ -238,51 +238,51 @@ bool checkReconstructedObject(GlobalMethodsClass const& gmc, LCCluster& thisClus
...
-  streamlog::out.addLevelName<marlin::ERROR2>();
-  streamlog::out.addLevelName<marlin::ERROR3>();
-  streamlog::out.addLevelName<marlin::ERROR4>();
-  streamlog::out.addLevelName<marlin::ERROR5>();
-  streamlog::out.addLevelName<marlin::ERROR6>();
-  streamlog::out.addLevelName<marlin::ERROR7>();
-  streamlog::out.addLevelName<marlin::ERROR8>();
-  streamlog::out.addLevelName<marlin::ERROR9>();
-  streamlog::out.addLevelName<marlin::SILENT>();
+  // streamlog::out.addLevelName<marlin::DEBUG>();
```
Comment (andresailer) [inline source/Tests/include/TestUtilities.hh]: Is this no longer necessary? Are all the log levels no already defined?

Comment (rete) [reply source/Tests/include/TestUtilities.hh]: Indeed it is not anymore. 

See https://github.com/iLCSoft/iLCUtil/pull/14/files#diff-3bf289d9dca0cee6235c43ccae938a15R600

This is called on logger construction

Comment (andresailer) [reply source/Tests/include/TestUtilities.hh]: Then please get completely rid of this function instead of commenting all the lines.

**Comment by andresailer** [comment]
In view of iLCSoft/iLCUtil#16, can this be closed?

**Comment by rete** [comment]
@andresailer Yes, I will open a new pull request later when iLCUtil is updated.

**Pull request closed without merging**